tests/e2e/config.go

```diff
@@ -243,11 +243,12 @@ func DefaultTestConfig() TestConfig {
 
 func DemocracyTestConfig(allowReward bool) TestConfig {
 	consumerGenChanges := ".app_state.ccvconsumer.params.blocks_per_distribution_transmission = \"20\" | " +
-		".app_state.gov.voting_params.voting_period = \"10s\" | " +
+		".app_state.gov.params.voting_period = \"10s\" | " +
```

Comment:
Gov props weren't executed due to this typo.

Reply:
Seems bad that small things like this break things (particularly that this seems to break them silently..), good catch!